src/http/caddy.rs

```diff
@@ -0,0 +1,131 @@
+use crate::utils::project_directory::get_rymfony_project_directory;
+
+use regex::Regex;
+use std::path::PathBuf;
+use std::fs;
+use std::process::Command;
+use std::process::Stdio;
+#[cfg(not(target_os="windows"))]
+use std::os::unix::fs::PermissionsExt;
+
+#[cfg(not(target_os="windows"))]
+use runas::Command as SudoCommand;
+
+const CADDY_VERSION_REGEX: &'static str = r"^v(2\.\d+\.\d+) ";
+
+#[cfg(target_os="windows")]
+const CADDY_BIN_FILE: &'static str = "caddy.exe";
+
+#[cfg(not(target_os="windows"))]
+const CADDY_BIN_FILE: &'static str = "caddy";
+
+pub(crate) const CADDYFILE: &'static str = "
+{
+    {{ debug }}debug
+    log {{ log_file }}
+    {{ use_tls }}local_certs
+}
+
+{{ host }}:{{ http_port }} {
+    root * {{ document_root }}
+
+    encode gzip
+
+    {{ use_tls }}tls internal
```

Comment:
FYI, the `local_certs` global option is essentially the same as having set `tls internal` on all your sites. Doesn't really make sense to set both. The global option is a shortcut for when testing/debugging to just make _all_ the configured sites use the local CA. `tls internal` is per-site.

Reply:
Good point 👍 Since every project using Rymfony will have its own "personal" caddy config, it's indeed not configuring multiple sites, so I'll remove the duplicate 👍